doc/SpecifyingApplications/CreatingSequenceDiagrams/CreatingSequenceDiagrams.md

```diff
@@ -0,0 +1,160 @@
+* when the diagram is exported to *.png*, the created picture file will be named with the string provided here
+
+**2: Diagram Title = ForwardingName**
+* the title is what will be shown as title in the rendered diagram
+* here, the name of the forwarding has to be provided
+* to come up with a proper name, it is advised to read [Structure of Internal _ForwardingNames_](../../ElementsApplicationPattern/Names/StructureOfInternalForwardingNames/StructureOfInternalForwardingNames.md)
+
+**3. Participants = diagram nodes**
+* the participant block is used for easier editing of the diagram
+* for each node in the diagram create a participant
+  * format: `participant "<text to be displayed in the diagram>" as <alias>`
+  * the `"<text>"` part contains what is shown in the rendered diagram, the `<alias>` is the reference to the node in the *.plantuml* raw view
+* the participant ordering determines the order in which the nodes are displayed in the diagram
+* typically, the first participant is the iniator of the request
+  * some examples for the `"<text>"` are "Admin", "/v1/embed-yourself", "external", "cyclic", a specific other application's (abbreviated) name or a combination of those
+  * chose the alias according to the node type, this can e.g. be a requestor or subscriber
+* if a node refers to a certain service it should be provided in the following format: `<AppAbbreviation>://<operation-name>`, e.g. *"NP://v1/listen-to-controller"* or *"MWDI://v1/provide-list-of-connected-devices"*
+  * including the AppAbbreviation allows to directly see to which application the operation server or operation client belongs to
+  * the service from the ServiceList for which the diagram was created, typically is the operation server, whereas other operation-names included in the diagram called due to that operation server, are the operation clients (either from the same app or a 3rd app)
+
+**4. Activation of the Forwarding**
+* add a connection between the operation server and it's preceding participant
+* then activate the operation server to indicate over which process steps the forwarding is active
+* once the forwarding is finished, deactivate the operation server participant
+* in the rendered diagram the activation is shown as a white vertical rectangle 
+
+**5. Notes Boxes**
+* sometimes additional information needs to be provided, e.g. if there are if-then-else conditions, which cannot be properly modelled in PlantUML
+* encapsulate those notes in `note over <participant>` and `end note` statements
+* the notes are shown as yellow boxes
+
+**6. Connections between Participants**
+* the connections between the participants show how the communication between the participants looks like
+* a sent request is modelled as `<caller participant> -> <called participant>: {input parameters}`
+  * it is shown as a solid line
+  * the parameters are optional, if there are none, leave out everything starting at the ":"

```

Comment:
Could you please change from
"the parameters are optional, if there are none, leave out everything starting at the ":""
to 
"the parameters are optional, and just for visualizing, whether all the required input values are available at this point of the sequence. If there are none, leave out everything starting at the ":"